certified-connectors/Asite (KSA)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1163 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Asite (KSA)",
+    "description": "Transform how your teams work together with the Asite connector, the ultimate collaboration solution to build connections between business systems.",
```

Comment:
@asitedeveloper It is possible to add in the description which region the connector belongs to? something like it is for KSA region?

Reply:
@sriyen-msft We have mentioned region in description.

Reply:
@asitedeveloper Can you please let us know where the "KSA region" is added in the connector's description?